### ISSUE TITLE
[1.10.2] Sonar Core & EnderIO issue

### ISSUE THREAD

**Issue by SnowShock35** [SonarSonic/Sonar-Core#19]
Hi,

I believe to have found a potential bug in Sonar Core. Here are the details. . . 

### Important Information

- Minecraft 1.10.2
- Minecraft Forge 12.18.3.2185
- Sonar Core 3.2.1
- Other mods:
  - Flux Networks 1.1.1
  - Ender IO 3.1.1.56

### Issue Description

When placing a Ender IO conduit on to a Flux Point from Flux Networks the game crashes to desktop. I imagine this is something that would have to be talked through with the EIO devs or might even be fixable without. The weird thing is that the crash wasn't consistent. It was a weird edge case that I've yet to be able to reproduce. 

### Issue Media

- [Crash Report](http://pastebin.com/NZH9XMKd)

**Comment by isots-code** [comment]
I'm also having this problem, although mine's quite strange, as it errored when I placed a Storage Crate from Actually Addictions but didn't crash, but I couldn't open it. Then I swapped for an Ender Chest and just the client crashed, my server stayed up, it didn't even complain. Here are some logs.

MC ver: 1.10.2
Forge: 12.18.3.2215
FTB SkyFactory 3 (v.3.0.5 modified- added MmmMmmMmmMmm (testdummy) an ProjectE) 

Log (without and with the crash): http://pastebin.com/bfng50u7
Crash report: http://pastebin.com/35iP3xuB
